README.md

```diff
@@ -1,2 +1,74 @@
-# react-template-1
-React project setup using create-react-app cli
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Air Pollution Tracker 🧮
+
+![cover1](./src/images/ap1.png) ![cover1](./src/images/ap2.png) ![cover1](./src/images/ap4.png) ![cover1](./src/images/ap3.png)
+
+> A web application that provides current air pollution data for all of the 56 capitals of the American continent!
+
+- Uses React components, props, life cycle methods
+- React Router.
+- Connect React and Redux.
+- Handle events in a React app.
+- Uses React Testing Library.
+- Uses Redux store, actions and reducers in React.
+
+## Built With
+
+- Air Pollution API from [OpenWeather](https://openweathermap.org/)
+- Javascript
+- create-react-app
+- React Router
+- React Testing Library 
+- Redux for state MGMT
+
+## See it live
+
+- [CHECK IT LIVE ON NETLIFY]()
```

Comment:
- [ ] Kindly ensure you correctly add your demo link

  ![live-jr](https://user-images.githubusercontent.com/36542986/176770738-f975d7c4-f99f-4d60-ba7d-089178bbbd80.jpg)